sdk/confidentialledger/Azure.Security.ConfidentialLedger/api/Azure.Security.ConfidentialLedger.netstandard2.0.cs

```diff
@@ -27,12 +27,13 @@ public partial class ConfidentialLedgerClient
         public virtual System.Threading.Tasks.Task<Azure.Response> GetTransactionStatusAsync(string transactionId, Azure.RequestOptions options = null) { throw null; }
         public virtual Azure.Response GetUser(string userId, Azure.RequestOptions options = null) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response> GetUserAsync(string userId, Azure.RequestOptions options = null) { throw null; }
-        public virtual Azure.Response PostLedgerEntry(Azure.Core.RequestContent content, string subLedgerId = null, Azure.RequestOptions options = null) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Response> PostLedgerEntryAsync(Azure.Core.RequestContent content, string subLedgerId = null, Azure.RequestOptions options = null) { throw null; }
+        public virtual Azure.Security.ConfidentialLedger.PostLedgerEntryOperation PostLedgerEntry(Azure.Core.RequestContent content, string subLedgerId = null, bool waitForCompletion = true, Azure.RequestOptions options = null) { throw null; }
```

Comment:
We need to decide if we keep using the Start prefix for SLROs.

Reply:
In previous discussions, we were leaning towards not using the Start prefix. Are there some new thoughts around keeping it?

Reply:
I got confused by the usage samples. Given the default for waitForCompletion is true, I think we should not use the Start prefix. The usage samples pass true explicitly and so I thought the default is not waiting.